service-base/src/main/java/org/eclipse/hono/service/tenant/TenantAmqpEndpoint.java

```diff
@@ -37,7 +37,7 @@
  * messages representing requests and sends them to an address on the vertx event bus for processing. The outcome is
  * then returned to the peer in a response message.
  */
-public final class TenantAmqpEndpoint extends RequestResponseEndpoint<ServiceConfigProperties> {
+public class TenantAmqpEndpoint extends RequestResponseEndpoint<ServiceConfigProperties> {
```

Comment:
Please make the `getName()` method final, too.

Reply:
Same as above.

Reply:
Done